pkg/core/app/app.go

```diff
@@ -100,21 +103,33 @@ func (a *App) ContainerIPv4Addr() string {
 
 func (a *App) SetupDocker() error {
 	var err error
-	cont, net, err := ParseDockerCmd(a.cmd)
-	if err != nil {
-		utils.LogError(a.logger, err, "failed to parse container name from given docker command", zap.String("cmd", a.cmd))
-		return err
-	}
-	if a.container == "" {
-		a.container = cont
-	} else if a.container != cont {
-		a.logger.Warn(fmt.Sprintf("given app container:(%v) is different from parsed app container:(%v)", a.container, cont))
-	}
 
-	if a.containerNetwork == "" {
-		a.containerNetwork = net
-	} else if a.containerNetwork != net {
-		a.logger.Warn(fmt.Sprintf("given docker network:(%v) is different from parsed docker network:(%v)", a.containerNetwork, net))
+	if a.kind == utils.DockerRun {
```

Comment:
Use switch case instead of "if else" for checking dockerRun or dockerStart.

Reply:
Refactored